### PR TITLE
Smudgeplot

### PR DESCRIPTION
**Pull request by RomainFeron**
This pull request adds a recipe for [smudgeplot](https://github.com/KamilSJaron/smudgeplot), a tool for inference of ploidy and heterozygosity structure using whole genome sequencing data.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).

### CODE REVIEW

File: recipes/smudgeplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    sha256: 2445564d12a852dcd5ade521b522cabde0df23512687b4ab5b46f73f9b315608
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - make
+        - r-devtools
+        - r-base
+        - r-viridis
+        - r-argparse
```
Comment (dpryan79) [inline recipes/smudgeplot/meta.yaml]: Please remove the R packages from the `build:` section.

File: recipes/smudgeplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    host:
+        - python >=3.6
+        - r-base
+        - r-viridis
+        - r-argparse
+        - numpy
+        - scipy
```
Comment (dpryan79) [inline recipes/smudgeplot/meta.yaml]: Are the python packages actually needed for installation or just during run time?

File: recipes/smudgeplot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
+    commands:
+        - smudgeplot.py --version 2>&1 | grep 'Running'
+
+about:
+    home: https://github.com/KamilSJaron/smudgeplot
+    license: Apache-2.0
```
Comment (dpryan79) [inline recipes/smudgeplot/meta.yaml]: I think the license is part of the tarball, so remove the license you manually added and add a `license_file: LICENSE` line.

File: recipes/smudgeplot/meta.yaml (hunk 2 above)
Comment (dpryan79) [reply recipes/smudgeplot/meta.yaml]: OK, I guess I was surprised to see an R script used for installation rather than `R CMD install`.

Comment (RomainFeron) [reply recipes/smudgeplot/meta.yaml]: Indeed. I will transfer this comment to the author, I think they want to improve the install process anyway

**Pull request merged**